app/api/ada/index.js

```diff
@@ -1176,7 +1191,11 @@ export default class AdaApi {
         derivation_scheme: cryptoAccount.derivation_scheme
       };
 
-      const _saveAsAdaAddresses = async _request => {
+      const addresses = [];
+      const indices = [];
+      const _saveAsAdaAddresses = async ({ accountIndex, addrs, offset, addressType}) => {
+        addresses = [...addrs];
```

Comment:
Don't think this will work. You have to append to `addresses` every time, not override it.